requirements.txt

```diff
@@ -4,4 +4,4 @@ pyroute2
 netifaces
 jsonschema
 jinja2
-git+https://opendev.org/openstack/charm-ops-sunbeam#egg=ops_sunbeam
+git+https://opendev.org/openstack/sunbeam-charms/#egg=ops-sunbeam&subdirectory=ops-sunbeam
```

Comment:
What is this change for?

Reply:
The charm-ops-sunbeam repository is retired and moved to the sunbeam-charms repository.